k8s/helm/metaflow/charts/metaflow-ui/templates/_helpers.tpl

```diff
@@ -97,22 +97,22 @@ Create the name of the service account to use
 
 {{- define "metaflow-ui.metadatadbEnvVars" -}}
 - name: MF_METADATA_DB_NAME
-  value: {{ .Values.metadatadb.name | quote }}
+  value: {{ .Values.uiBackend.metadatadb.name | quote }}
 - name: MF_METADATA_DB_PORT
-  value: {{ .Values.metadatadb.port | quote }}
+  value: {{ .Values.uiBackend.metadatadb.port | quote }}
 - name: MF_METADATA_DB_PSWD
-  value: {{ .Values.metadatadb.password | quote }}
+  value: {{ .Values.uiBackend.metadatadb.password | quote }}
 - name: MF_METADATA_DB_USER
-  value: {{ .Values.metadatadb.user | quote }}
-{{- if .Values.metadatadb.host }}
+  value: {{ .Values.uiBackend.metadatadb.user | quote }}
+{{- if .Values.uiBackend.metadatadb.host }}
 - name: MF_METADATA_DB_HOST
-  value: {{ .Values.metadatadb.host | quote }}
+  value: {{ .Values.uiBackend.metadatadb.host | quote }}
```

Comment:
Comment for posterity: should we ever support feeding these in via secrets and not pass them down from the values file? Not something we need to do now. Just something to keep in the back of our minds.

Reply:
I agree, users should still be able to pass in the values using environment values or a k8s secret using a values file, which are more secure options than just passing it on the command line. I will add a note to mention this in the comments section.

Reply:
added comment